doc/neps/nep-0036-fair-play.rst

```diff
@@ -26,31 +26,34 @@ troublesome, bothersome, and unacceptable.
 Motivation
 ----------
 
-We sometimes learn of NumPy versions modified and circulated by outsiders.
-These patched versions can cause problems for the NumPy community.
+Every so often, we learn of NumPy versions modified and circulated by outsiders.
+These patched versions can cause problems for the NumPy community
+(see, e.g., [#erf]_ and [#CVE-2019-6446]_).
+When issues like these arise, our developers waste time identifying
+the problematic release, locating alterations, and determining an
+appropriate course of action.
 
-- In December 2018, a `bug report
-  <https://github.com/numpy/numpy/issues/12515>`__ was filed against
-  `np.erf` -- a function that didn't exist in the NumPy distribution.
-  It came to light that a company had published a NumPy version with
-  an extended API footprint. After several months of discussion, the
-  company agreed to make its patches public, and we added a label to
-  the NumPy issue tracker to identify issues pertaining to that
-  distribution.
-
-- In another case, after a security issue (CVE-2019-6446) was filed
-  against NumPy, distributions put in their own fixes, most often by
-  changing a default keyword value. As a result the NumPy API was
-  inconsistent across distributions.
-
-When issues arise in cases like these, our developers waste time
-identifying the problematic release, locating alterations,
-and determining an appropriate course of action.
+In addition, packages on the Python Packaging Index are sometimes
+named such that users assume they are sanctioned or maintained by
+NumPy.  We wish to reduce the number of such incidences.
```

Comment:
```suggestion
NumPy.  We wish to reduce the number of such incidents.
```
I think "incident" is the appropriate word when referring to discrete events, but NBD either way - feel free to ignore.